### PR TITLE
Uvec package

### PR DESCRIPTION
**Pull request by brunozc**
Refactored the entire code with pre-commit

Added the feature to get the instalation path of the uvec

### CODE REVIEW

File: UVEC/get_path_uvec.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    """
+    with open(os.path.join(folder_path, "direct_url.json"), "r") as f:
+        data = json.load(f)
+
+        path_package = data["url"].split("file://")[1]
+
+        with open(os.path.join(folder_path, "top_level.txt"), "r") as f:
+            packages = f.read().splitlines()
+
+        return os.path.join(path_package, packages[0])
```
Comment (aronnoordam) [inline UVEC/get_path_uvec.py]: should this still be indented?

File: UVEC/get_path_uvec.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        - package_name (str): Name of the package to check.
+
+    Returns:
+        - bool: True if installed in editable mode, otherwise False.
+    """
+
+    if os.path.isfile(os.path.join(site.getsitepackages()[0], f"__editable__.{package_name}.pth")):
+        return True
+    else:
+        return False
```
Comment (aronnoordam) [inline UVEC/get_path_uvec.py]:
```suggestion
    return os.path.isfile(os.path.join(site.getsitepackages()[0], f"__editable__.{package_name}.pth")):
```

Comment (brunozc) [reply UVEC/get_path_uvec.py]: done

File: UVEC/get_path_uvec.py (hunk 1 above)
Comment (brunozc) [reply UVEC/get_path_uvec.py]: yes. fixed

**Review by aronnoordam — APPROVED** [review]

**Pull request merged**